project-docs/stumptown-writers-guide.md

```diff
@@ -0,0 +1,697 @@
+
+```yaml
+attributes:
+    element_specific:
+        - ../input/attributes/autocomplete.md
+        - ../input/attributes/name.md
+    global: /content/html/global_attributes
+```
+
+In this example, there are two element-specific attributes, found at "../input/attributes/autocomplete.md" and "../input/attributes/name.md", relative to the directory containing this front matter.
+
+Paths may be given as an absolute path from the stumptown root directory, or a path relative to the file containing the ingredient.
+
+**Note: I think we should remove `global`, since everyone always has it and it always has the same value.**
+
+Each element-specific attribute is described in its own Markdown file, with its own syntax.
+
+#### Element-specific HTML attribute syntax
+
+Here's an example HTML attribute file:
+
+```md
+---
+browser-compatibility: html.elements.audio.crossorigin
+specifications: https://html.spec.whatwg.org/multipage/media.html#attr-media-crossorigin
+---
+
+# `crossorigin`
+
+This enumerated attribute ...
+
+## Values
+
+### `anonymous`
+
+Sends a cross-origin request without a credential...
+
+### `use-credentials`
+
+Sends a cross-origin request with a credential...
+
+## Type
+
+String
+```
+
+The file may contain some YAML front matter, with the following two properties, both optional:
+
+-   `browser_compatibility`: syntax is the same as the `browser_compatibility` ingredient
+-   `specifications`: syntax is the same as the `specifications` ingredient.
+
+The H1 heading must be in `code format` and its content must be the name of the attribute:
+
+```md
+# `crossorigin`
+```
+
+Prose immediately after this describes the attribute.
+
+Some attributes, such as the `crossorigin` attribute above, can take special enumerated values. For such attributes, the description should be followed by an H2 with the text "Values":
+
+```
+## Values
+```
+
+If there is a "Values" H2, then it must be followed by one or more H3 headings, with text content in `code format`, whose text content is a possible value for the attribute.
+
+The text following this H3 describes that value.
+
+After the "Values" section there is a mandatory H2 "Type":
+
+```
+## Type
+```
+
+After this there is an string describing the type of this attribute. The following are valid values for this string:
+
+"String", "Number", "Boolean", "URL", "RegExp".
+
+### examples
+
+An object used to specify static or live samples to include in the page.
+
+**Note: this does not specify interactive examples, because we'd have to rewrite the live sample system if we wanted to do that.**
+
+It is given as an array of paths to directories. Each directory contains a single example:
+
+```yaml
+examples:
+    - examples/simple-example
+    - examples/audio-element-with-video-element
+    - examples/multiple-source-elements
+```
+
+**Note: it looks like this is using a different path convention from attributes. We should be consistent here.**
```

Comment:
I like this one. (without any "../" or "./")

Reply:
Filed as https://github.com/mdn/stumptown-content/issues/204.